Danki2/Assets/Scripts/Actor/ActorExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿public static class ActorExtensions
+{
+    private static string playerTag = "Player";
+    private static string enemyTag = "Enemy";
+    private static string friendlyTag = "Friendly";
```

Comment:
This class could be rename to `TagUtils` if the extension methods go and these strings can be public and readonly. You could add a method `public static boolean IsActor(GameObject gameObject)` that returns true if the game object has the tag "Player" or "Enemy". Then you could use this for the projectiles